mysite/account/views.py

```diff
@@ -0,0 +1,62 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.shortcuts import render, redirect
+from django.contrib.auth.models import User, auth
+# Create your views here.
+
+
+
+def home(request):
+    return render(request, 'home.html')
+
+
+def register(request):
+    if request.method == 'POST':
+        first_name = request.POST['first_name']
+        last_name = request.POST['last_name']
+        username = request.POST['username']
+        email = request.POST['email']
+        password = request.POST['password']
+        confirm_password = request.POST['confirm_password']
+
+        if password == confirm_password:
+            if User.objects.filter(username = username):
+                messages.info(request, 'Your name has existed')
+                return redirect(register)
+            
+            elif User.objects.filter(email = email):
+                messages.info(request, 'Your email has been used')
+
+                return redirect(register)
+            
+            else:
+                user = User.objects.create_user(username=username, password=password, 
+                                                email=email, first_name=first_name, last_name=last_name)
+                user.save()
+                return redirect('login')
+        else:
+            messages.info(request, 'Both passwords are not matching')
+            return redirect('register')
+    else:
+        return render(request, 'registeration.html')
+
+
+def login(request):
```

Comment:
how about if the user has logged in ? Try researching class base-view, and apply